dlt/cli/init_command.py

```diff
@@ -38,10 +41,16 @@
 from dlt.cli.exceptions import CliCommandException
 from dlt.cli.requirements import SourceRequirements
 
+
 DLT_INIT_DOCS_URL = "https://dlthub.com/docs/reference/command-line-interface#dlt-init"
 DEFAULT_VERIFIED_SOURCES_REPO = "https://github.com/dlt-hub/verified-sources.git"
 INIT_MODULE_NAME = "init"
 SOURCES_MODULE_NAME = "sources"
+SKIP_CORE_SOURCES_FOLDERS = [
```

Comment:
should we declare this in `dlt.sources`?

Reply:
I have moved it to the same location where the verified sources config is for now